source/mainnet/net/resources/glossary.rst

```diff
@@ -486,6 +486,13 @@ Private keys
 
 A large, randomly-generated, secret number that is used in cryptography and cryptocurrency to prove ownership of an account and sign transactions to send, spend, delegate, and stake CCDs. A wallet consists of a set of public addresses and private keys. Anyone can deposit cryptocurrency in a public address, but funds cannot be removed from an address without the corresponding private key.
 
+.. _glossary-qualified-authority:
+
+Qualified authority
```

Comment:
Was this just added now? I didn't see it before. 

It makes perfect sense to me, but I have no idea whether the definition is correct or not

Reply:
Ales has already reviewed Qualified authority. https://github.com/Concordium/concordium.github.io/pull/745